docs/quickstarts/ratify-high-availability.md

```diff
@@ -164,4 +164,28 @@ helm delete dapr --namespace dapr-system
 kubectl delete Component dapr-redis -n gatekeeper-system
 kubectl delete Secret ratify-dapr-signing-key -n gatekeeper-system
 helm delete gatekeeper -n gatekeeper-system
+```
+
+## Development Testing with Helmfile (build your own images)
```

Comment:
@binbin-li @susanshi should I move this out of here?

Reply:
That's good question. Do you think `CONTRIBUTING.md` would better fit it? @junczhu wonder which doc did you refer to for local building/testing?

Reply:
Actually since `CONTRIBUTING.md` could be shared by both HA and manual doc, I'm OK having it here.